app/views/qualifications/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render 'shared/form_errors_summary', record: @qualification %>
+
+<div class='govuk-grid-row'>
+  <div class='govuk-grid-column-two-thirds'>
+    <%= form_with model: @qualification, builder: GOVUKDesignSystemFormBuilder::FormBuilder, action: :edit, html: { novalidate: true } do |f| %>
+      <legend class='govuk-fieldset__legend govuk-fieldset__legend--xl'>
+        <h1 class='govuk-fieldset__heading'>
+          <span class='govuk-caption-xl'>Academic qualifications</span>
+          Add qualification
+        </h1>
+      </legend>
+
+      <%= f.govuk_text_field :type_of_qualification, label: { text: t('application_form.qualification_section.type.label')}, width: 20 %>
+      <%= f.govuk_text_field :subject, label: { text: t('application_form.qualification_section.subject.label')}, width: 20 %>
+      <%= f.govuk_text_field :institution, label: { text: t('application_form.qualification_section.institution.label')}, width: 20 %>
+      <%= f.govuk_text_field :grade, label: { text: t('application_form.qualification_section.grade.label')}, width: 20 %>
+      <%= f.govuk_text_field :year, label: { text: t('application_form.qualification_section.year.label')}, width: 20 %>
```

Comment:
The field widths are different — see prototype.

Reply:
I've just removed `width` since it is not specified in the prototype.